ros/lib/rbac_denied_exception.py

```diff
@@ -0,0 +1,8 @@
+class RBACDenied(Exception):
+    def __init__(self, message):
+        """
+        Raise this exception if the inventory service reports that you do not
```

Comment:
@apuntamb - do we need this exception class? 
The reason behind asking this question is doc string says that raise it if inventory service reports no permission to access. IIRC, ROS backend don't request any inventory API.

@beav, any thoughts?

Reply:
In addition, here also I didn't see @apuntamb raising this exception anywhere.

Reply:
@kgaikwad The motive is to raise a specific exception only for this `try block` [here](https://github.com/RedHatInsights/ros-backend/pull/80/files#diff-55bc290eed399bee5e1cb3fbc4494c7c218710af4a7adbc2c8cf3c5e92ad6e21R96-R112). I do not want to catch it in a generic `Exception` class because in that case any exception even though with its specific abort statement will be caught here. If you see I'm raising this RBACDenied on line-112.

Should I not use a try-except instead and abort the request where necessary? In that case I suppose I need not create a custom class. What do you think?

Reply:
> If you see I'm raising this RBACDenied on line-112.

From [line-112]( https://github.com/RedHatInsights/ros-backend/pull/80/files#diff-55bc290eed399bee5e1cb3fbc4494c7c218710af4a7adbc2c8cf3c5e92ad6e21R112), you have used `except` block.

The except block lets you handle the error. To raise an exception, one needs to use the `raise` keyword.